packages/utils/src/lib/format-command-log.integration.test.ts

```diff
@@ -0,0 +1,46 @@
+import path from 'node:path';
+import { describe, expect, it } from 'vitest';
+import { removeColorCodes } from '@code-pushup/test-utils';
+import { formatCommandLog } from './format-command-log.js';
+
+describe('formatCommandLog', () => {
+  it('should format simple command', () => {
+    const result = removeColorCodes(
+      formatCommandLog('npx', ['command', '--verbose']),
+    );
+
+    expect(result).toBe('$ npx command --verbose');
+  });
+
+  it('should format simple command with explicit process.cwd()', () => {
+    const result = removeColorCodes(
+      formatCommandLog('npx', ['command', '--verbose'], process.cwd()),
+    );
+
+    expect(result).toBe('$ npx command --verbose');
+  });
+
+  it('should format simple command with relative cwd', () => {
+    const result = removeColorCodes(
+      formatCommandLog('npx', ['command', '--verbose'], './wololo'),
+    );
+
+    expect(result).toBe(
+      `${path.join(process.cwd(), 'wololo')} $ npx command --verbose`,
+    );
+  });
```

Comment:
This is the opposite of what I would expect :sweat_smile: To reduce verbosity, the relative path should be printed, not the full path. In fact, if `cwd` is an absolute path, then it should be reduced to a relative path. Expanding a relative path to an absolute path (what's being tested here) is counter-productive.

Reply:
I see, I'll switch the behavior.